src/Calculator/Calculator.vcxproj

```diff
@@ -227,6 +226,12 @@
   </PropertyGroup>
   <PropertyGroup>
     <GenerateProjectSpecificOutputFolder>true</GenerateProjectSpecificOutputFolder>
+    <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
```

Comment:
Probably left over from generating the appx, can we remove this block of new code?